frontend/src/components/navigation/MenuToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import { useLocalization } from "@fluent/react";
+import { CloseIcon, MenuIcon } from "../Icons";
+import styles from "./MenuToggle.module.scss";
+
+export const MenuToggle = ({ ...props }): JSX.Element => {
+  const { l10n } = useLocalization();
+  const { toggleState } = props;
+
+  const ToggleMenuOpen = () =>
+    MenuIcon({
+      alt: l10n.getString("menu-toggle-open"),
+    });
```

Comment:
Wow, I must have been speeding through this (roughing it out) - it just made sense that the icons can just be called as JSX elements since that is what they return... great catch. I went ahead and cleaned this up - I suspect I did this in a few other places too.